examples/read_pem_files/main.rs

```diff
@@ -0,0 +1,63 @@
+use config::{Config, Format, Map, Value, ValueKind, FileStoredFormat};
```

Comment:
Need to add another import for supporting the custom error:

```suggestion
use config::{Config, Format, Map, Value, ValueKind, FileStoredFormat};
use std::io::{Error, ErrorKind};
```